common/changes/@microsoft/api-documenter/clean-up-config-file-loading_2023-06-11-08-57.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "type": "none",
```

Comment:
Why is the change type `none`?

Reply:
The initial PR hasn't been released yet.